12-hooks-in-react/src/components/Search.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from "react";
+
+const Search = () => {
+  const [term, setTerm] = useState("");
```

Comment:
I think you can use null or leave it blank? That's better so if u plan to check if term has a value in the future it's easier to check

Reply:
I believe that setting an empty string is better than null, because that will give the other developers the idea that the supposed value in the future will be a string.

Reply:
Ok accepted. But do note that it's not always needed to be a string, this is important for form validation since it's easier to validate if a user hasn't entered anything using null compared to an empty string